win32/Makefile.a64

```diff
@@ -29,6 +29,7 @@ WFLAGS  = \
 	-D_CRT_NONSTDC_NO_DEPRECATE \
 	-DUNALIGNED_OK \
 	-DUNALIGNED64_OK \
+	-DARM_NEON_HASLD4 \
 	-D_ARM64_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE=1 \
 	-DARM_FEATURES \
```

Comment:
Maybe should keep these in some kind of lexical order... Usually underscore comes before anything else though...